### PR TITLE
i added kubernetes driver information to drivers section

### PR DESCRIPTION
**Pull request by developer-guy**
i saw there is no available kubernetes driver in documentation but in driver-opts section there is , so i added it to drivers section  too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -387,6 +387,8 @@ Sets the builder driver to be used. There are two available drivers, each have t
 
 - `docker-container` - Uses a buildkit container that will be spawned via docker. With this driver, both building multi-platform images and exporting cache are supported. However, images built will not automatically appear in `docker images` (see [`build --load`](#--load)).
 
+- `kubernetes` - Uses a kubernetes pods. With this driver , you can spin up pods with defined buildkit container image to built your images.
```
Comment (tonistiigi) [inline README.md]: `s/built/build/`

Comment (developer-guy) [reply README.md]: thanks @tonistiigi , i already fixed it.

**Comment by tonistiigi** [comment]
@developer-guy Please also squash the commits. https://github.com/docker/buildx/blob/master/.github/CONTRIBUTING.md#review

**Comment by developer-guy** [comment]
> @developer-guy Please also squash the commits. [/.github/CONTRIBUTING.md@`master`#review](https://github.com/docker/buildx/blob/master/.github/CONTRIBUTING.md#review) [](/docker/buildx/blob/HEAD@{2020-03-28T18:48:58Z}/.github/CONTRIBUTING.md#review)

done. @tonistiigi

**Pull request merged**